### ISSUE TITLE
[BUG] TFT forecast negative values although there are no negative target in training data

### ISSUE THREAD

**Issue by xlsi** [unit8co/darts#2154]
**Describe the bug**
TFT forecasts negative value although there are no negative targets in training data

**To Reproduce**
Deterministic forecast, with MAE as metric.
Targets are first scaled with darts.dataprocessing.transformers.Scaler
Best params: {'input_chunk_length': 22, 'output_chunk_length': 10, 'hidden_size': 256, 'lstm_layers': 3, 'num_attention_heads': 32, 'full_attention': False, 'dropout': 0.07764449837435405, 'lr': 0.0005731156347991463}

**System (please complete the following information):**
 - Python version: [3.9]
 - darts version [0.27.1]
